logstash-core/lib/logstash/modules/kibana_config.rb

```diff
@@ -28,26 +28,22 @@ def dashboards
     end
   end
 
-  def kibana_index_hacks
-    # Copied from libbeat/dashboards/importer.go
-    # CreateKibanaIndex creates the kibana index if it doesn't exists and sets
-    # some index properties which are needed as a workaround for:
-    # https://github.com/elastic/beats-dashboards/issues/94
-    # with kibana 5.4.0 this hack failed to be applied.
-    ha = '{"settings": {"index":{"mapping":{"single_type": false}}}}'
-    hb = '{"search": {"properties": {"hits": {"type": "integer"}, "version": {"type": "integer"}}}}'
+  def index_pattern
+    pattern_name = "#{@name}-*"
+    default_index_json = '{"defaultIndex": "#{pattern_name}"}'
+    default_index_content_id = @settings.fetch("index_pattern.kibana_version", "5.4.0") # make this 5.5.0
```

Comment:
well i think its ok to leave for now for testing. when we merge the adjustments for the 5.X branch we should change it then.
We can change it as a bug fix in master at any time.